musicAlbum/musicalbum.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../item'
+
+class MusicAlbum
```

Comment:
- [ ] Kindly inherit from `Item` class. As it is shown in the UML class diagram `MusicAlbum` class is a child class of `Item` class.

Reply:
Thank you for pointing this one out. I have resolved it.